src/components/v5/common/ActionSidebar/partials/forms/AdvancedPaymentForm/hooks.ts

```diff
@@ -119,23 +116,9 @@ export const useAdvancedPayment = (
         ? ActionTypes.ACTION_EXPENDITURE_PAYMENT
         : ActionTypes.MOTION_EXPENDITURE_PAYMENT,
     getFormOptions,
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    transform: useCallback(
-      pipe(
-        mapPayload((payload: AdvancedPaymentFormValues) => {
-          return getCreatePaymentDialogPayload(
-            colony,
-            {
-              fromDomainId: payload.from,
-              payments: [],
-              annotation: payload.description,
-              motionDomainId: payload.createdIn,
-            },
-            networkInverseFee,
-          );
-        }),
-      ),
-      [colony, networkInverseFee],
-    ),
+    transform: mapPayload((payload: AdvancedPaymentFormValues) => {
+      // @TODO: Add a helper function mapping form values to action payload
+      return payload;
```

Comment:
hmmm will this work or is it OK since we don't have advanced payments in yet?

Reply:
It has never worked before, and as you pointed we don't have this action enabled yet. The problem here was that it was incorrectly depending on the old payment dialog helper, so the payload was wrong regardless.